tools/c7n_mailer/requirements.txt

```diff
@@ -14,4 +14,5 @@ ruamel.yaml<0.15 # rq.filter: <0.15
 six==1.11.0
 redis
 datadog
-slackclient
\ No newline at end of file
+slackclient
+sendgrid
```

Comment:
c7n_azure also becomes a dependency, and these should also go into setup.py. we haven't to date had a hard dep on c7n from the mailer (originally to facilitate lambda usage) but seems fine here (cli usage).

Reply:
eek, meant to fix setup.py then completely forgot somehow. tx.

Reply:
Hmm.  We can't depend on c7n_azure anyway as we haven't published the package yet and that would be a big dependency for people that didn't need it in mailer (Azure SDK).

Maybe I can try to opportunistically import the c7n_azure import in our azure_queue_processor and not depend on it?  @kapilt

Reply:
I went ahead and published c7n_azure from trunk as 0.1, so it can work fwiw.  The conditional import setup seems like a good workaround to avoid the large download if not needed. The azure part in the readme should note/doc the additional install step. Thanks

Reply:
Excellent thanks - updated import strategy, setup.py, docs, etc.